crowdstrike/table_crowdstrike_intel_actor.go

```diff
@@ -0,0 +1,193 @@
+					Operators: []string{"="},
+				},
+				{
+					Name:      "active",
+					Require:   plugin.Optional,
+					Operators: []string{"="},
+				},
+				{
+					Name:      "actor_type",
+					Require:   plugin.Optional,
+					Operators: []string{"="},
+				},
+				{
+					Name:      "created_date",
+					Require:   plugin.Optional,
+					Operators: []string{">", ">=", "=", "<", "<="},
+				},
+				{
+					Name:      "first_activity_date",
+					Require:   plugin.Optional,
+					Operators: []string{">", ">=", "=", "<", "<="},
+				},
+				{
+					Name:      "last_activity_date",
+					Require:   plugin.Optional,
+					Operators: []string{">", ">=", "=", "<", "<="},
+				},
+				{
+					Name:      "last_modified_date",
+					Require:   plugin.Optional,
+					Operators: []string{">", ">=", "=", "<", "<="},
+				},
+			},
+		},
+		Columns: []*plugin.Column{
+			{Name: "active", Description: "If this actor is still active.", Type: proto.ColumnType_BOOL},
+			{Name: "actor_type", Description: "The type of actor.", Type: proto.ColumnType_STRING},
+			{Name: "capability", Description: "The actor's capability.", Type: proto.ColumnType_JSON},
+			{Name: "created_date", Description: "The creation date.", Type: proto.ColumnType_TIMESTAMP, Transform: transform.FromField("CreatedDate").Transform(int642TimestampTransformer)},
+			{Name: "description", Description: "A description of the actor.", Type: proto.ColumnType_STRING},
+			{Name: "ecrime_kill_chain", Description: "eCrime kill chain fields.", Type: proto.ColumnType_JSON},
+			{Name: "entitlements", Description: "Entitlements of the actor.", Type: proto.ColumnType_JSON},
+			{Name: "first_activity_date", Description: "Date when first activity was detected.", Type: proto.ColumnType_TIMESTAMP, Transform: transform.FromField("FirstActivityDate").Transform(int642TimestampTransformer)},
+			{Name: "group", Description: "The actor's group.", Type: proto.ColumnType_JSON},
+			{Name: "id", Description: "The actor's ID.", Type: proto.ColumnType_INT},
+			{Name: "image", Description: "URL to the image of the Actor.", Type: proto.ColumnType_JSON},
+			{Name: "kill_chain", Description: "Kill chain fields.", Type: proto.ColumnType_JSON},
+			{Name: "known_as", Description: "The actor's alias.", Type: proto.ColumnType_STRING},
+			{Name: "last_activity_date", Description: "Date of last activity.", Type: proto.ColumnType_TIMESTAMP, Transform: transform.FromField("LastActivityDate").Transform(int642TimestampTransformer)},
+			{Name: "last_modified_date", Description: "Date when this actor was last modified.", Type: proto.ColumnType_TIMESTAMP, Transform: transform.FromField("LastModifiedDate").Transform(int642TimestampTransformer)},
+			{Name: "motivations", Description: "The actor's motivations.", Type: proto.ColumnType_JSON},
+			{Name: "name", Description: "The actor's name.", Type: proto.ColumnType_STRING},
+			{Name: "notify_users", Description: "Notified users.", Type: proto.ColumnType_BOOL},
```

Comment:
Is this description accurate? Should it instead be something like `True if users have been notified.` (not sure if this is actually what the field represents, but seems like it indicates if users have been notified or will be notified in case of an event).

Reply:
Please see comment above for the `notified_users` column